pull/context.go

```diff
@@ -42,6 +45,12 @@ type Context interface {
 	// string is formated as "<owner>/<repository>#<number>"
 	Locator() string
 
+	// Owner returns the owner of the repo that the pull request targets.
+	Owner() string
```

Comment:
Because this type is associated with a pull request, I'd call these `RepositoryOwner()` and `RepositoryName()` to make things more specific (or `RepoOwner` and `RepoName` if you want something shorter.)